projects/packages/masterbar/src/class-main.php

```diff
@@ -29,7 +29,10 @@ public static function init() {
 
 		new Admin_Color_Schemes();
 
-		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+		remove_filter( 'pre_option_wpcom_admin_interface', 'wpcom_admin_interface_pre_get_option' );
+		$is_wp_admin_interface = get_option( 'wpcom_admin_interface' ) === 'wp-admin';
+		add_filter( 'pre_option_wpcom_admin_interface', 'wpcom_admin_interface_pre_get_option', 10 );
```

Comment:
`wpcom_admin_interface_pre_get_option` doesn't exist on Jetpack sites, so I wonder we should add another `function_exists` check here.

Not sure if this file is loaded on Jetpack sites (maybe by the `admin-menu` endpoint?) though.

Reply:
Added a check just in case.